packages/nc-gui/utils/urlUtils.ts

```diff
@@ -20,8 +20,7 @@ export const replaceUrlsWithLink = (text: string): boolean | string => {
 }
 
 export const isValidURL = (str: string) => {
-  return isURL(`${str}`)
-}
+  return isURL(`${str}`, { protocols : ['http', 'https', 'ftp', 'file'] })
 
```

Comment:
Missing `}` here

Reply:
even so, adding url with file is not clickable.

Reply:
To make it work, you have to allow empty host name option for file protocol.

```js
  return (
    isURL(`${str}`, { protocols: ['http', 'https', 'ftp'] }) ||
    isURL(`${str}`, { protocols: ['file'], require_host: false })
  )
```